librocksdb-sys/rocksdb_lib_sources.txt

```diff
@@ -136,6 +142,12 @@ options/options.cc
 options/options_helper.cc
 options/options_parser.cc
 port/port_posix.cc
+port/win/env_default.cc
+port/win/env_win.cc
+port/win/io_win.cc
+port/win/port_win.cc
+port/win/win_logger.cc
+port/win/win_thread.cc
```

Comment:
These files isn't needed for unix like OS. They're added manually in build.rs.

Reply:
Ah! Removed.